src/components/addDogForm.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { saveDog } from "../services/dogService";
+import { Form, Field } from "react-final-form";
+import DatePicker from "react-datepicker";
+import FormTextField from "../components/common/formTextField"
+import FormCheckBox from "./common/formCheckBoxField"
+import FormOptionField from "./common/formOptionField"
+import { LookupDataContext } from "../lookupDataProvider"
+
+class AddDogForm extends Component {
+  state = {
+    lookupData: null
+  };
+
+  async componentWillMount() {
+    console.log(this.context.lookupData)
+    this.setState({
+      lookupData: this.context.lookupData
+    });
+  }
+
+  onSubmit = async values => {
+    await saveDog(values);
+
+    this.props.history.push("/add");
+    window.location = "/";
+  }
+
+  DatePickerAdapter = ({ input: { onChange, value }, ...rest }) => (
+    <DatePicker selected={value} onChange={date => onChange(date)} {...rest} />
+  );
+
+  render() {
+    return (
+      <React.Fragment>
+        <Form
+          DatePickerAdapter={this.DatePickerAdapter}
+          onSubmit={this.onSubmit}
+          // TODO: use initial values when display edit dog profile page
+
+          initialValues={{
+            AcuteAdoption: false,
+            RequiresExperiencedOwner: false,
+            Castrated: false,
+            Dewormed: false,
+            Chipped: false,
+            RequiresGarden: false
+          }}
+          render={({ handleSubmit, form, submitting, pristine, values }) => (
+            <form onSubmit={handleSubmit}>
+              <div>
+                <FormTextField name={'Name'} label={'Meno'} />
```

Comment:
Maly tip - ak sa do props priradzuje string, staci to napisat takto:

```suggestion
                <FormTextField name="Name" label="Meno" />
```

Reply:
vdaka za tip :)